pkg/node/node.go

```diff
@@ -1313,7 +1313,7 @@ func (b *Bee) Shutdown() error {
 	}
 
 	var wg sync.WaitGroup
-	wg.Add(10)
+	wg.Add(8)
```

Comment:
oh wow

Reply:
yet to learn loving them magic numbers... :)

Reply:
Agreed.  It seems to me it would be better for each section to add its own number rather than trying to keep an accurate total here.